static/src/js/components/MetadataForm/MetadataForm.jsx

```diff
@@ -96,11 +95,11 @@ const MetadataForm = () => {
 
   useEffect(() => {
     // If fieldName was pulled from the URL, set it to the focusField
-    setFocusField(fieldName)
+    if (fieldName) setFocusField(fieldName)
 
     // If a fieldName was pulled from the URL, then remove it from the URL. This will happen after the field is focused.
     if (fieldName && sectionName) navigate(`/drafts/${draftType}/${conceptId}/${sectionName}`, { replace: true })
-  }, [fieldName])
+  }, []) // Should only do this redirect on page load
```

Comment:
Comment seems out of place. Not sure if its necessary. If you want to keep it, maybe it could go above the `useEffect`.